net/transmission/Portfile

```diff
@@ -69,21 +56,19 @@ destroot {
         ${destroot}${prefix}/bin
 
         xinstall -m 644 -W ${worksrcpath}/daemon \
-        transmission-daemon.1 transmission-remote.1 \
+        transmission-daemon.1 \
+        ${destroot}${prefix}/share/man/man1
+        
+        xinstall -m 644 -W ${worksrcpath}/utils \
+        transmission-remote.1 \
         ${destroot}${prefix}/share/man/man1
    }
 }
 
-if {([vercmp $xcodeversion 7.0] < 0) || ([vercmp ${macosx_deployment_target} 10.9] < 0)} {
+if {([vercmp $xcodeversion 8.0] < 0) || ([vercmp ${macosx_deployment_target} 10.10] < 0)} {
```

Comment:
Please fix the warning message below to match the changed versions.
Ideally make them local variables, so you do not have to hardcode the same versions in two places.